Userland/Utilities/find.cpp

```diff
@@ -563,17 +565,13 @@ static void walk_tree(FileData& root_data, Command& command)
     closedir(dir);
 }
 
-int main(int argc, char* argv[])
+ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    LexicalPath root_path(parse_options(argc, argv));
+    LexicalPath root_path(parse_options(arguments.argc, arguments.argv));
     String dirname = root_path.dirname();
     String basename = root_path.basename();
 
-    int dirfd = open(dirname.characters(), O_RDONLY | O_DIRECTORY | O_CLOEXEC);
-    if (dirfd < 0) {
-        perror(dirname.characters());
-        return 1;
-    }
+    int dirfd = TRY(Core::System::open(dirname.characters(), O_RDONLY | O_DIRECTORY | O_CLOEXEC));
```

Comment:
You can drop the `.characters()` here, as `Core::System::open()` takes a `StringView` :)

Reply:
Very cool, haven't even thought of checking if the signature for the wrapper changed!